content/MoM/mom1e/tiles.ini

```diff
@@ -24,6 +36,11 @@ pps=146.28571
 [TileSideFrontPorch]
 image="../ffg/img/Tile_Front_Porch_MAD01.dds"
 
+[TileSideFrontYard]
+image="img/Tile_Front_Yard_MAD01.jpg"
+aspect=1.5
```

Comment:
This is only required for the imported textures because they are the wrong aspect.  It would be better to have the image at the correct aspect (3x2) and not have this line.